packages/browser/src/sdk.ts

```diff
@@ -90,14 +91,25 @@ export function init(options: BrowserOptions = {}): Client | undefined {
   const shouldDisableBecauseIsBrowserExtenstion =
     !options.skipBrowserExtensionCheck && checkAndWarnIfIsEmbeddedBrowserExtension();
 
+  let defaultIntegrations =
+    options.defaultIntegrations == null ? getDefaultIntegrations(options) : options.defaultIntegrations;
+
+  /* rollup-include-development-only */
+  if (options.spotlight) {
```

Comment:
How hard would it be to fill this value from the env variables? I'm okay with trying all variants like `PUBLIC_SENTRY_SPOTLIGHT` or `VITE_SENTRY_SPOTLIGHT` etc.

Happy to do this in a follow up too.